pkg/policyconditions/conditions.go

```diff
@@ -136,6 +154,12 @@ func SetPolicyFailedToConfigure(conditions *nmstate.ConditionList, message strin
 		nmstate.NodeNetworkConfigurationPolicyConditionConfigurationProgressing,
 		"",
 	)
+	conditions.Set(
+		nmstate.NodeNetworkConfigurationPolicyConditionIgnored,
+		corev1.ConditionFalse,
+		nmstate.NodeNetworkConfigurationPolicyConditionConfigurationProgressing,
```

Comment:
This should be `nmstate.NodeNetworkConfigurationPolicyConditionFailedToConfigure`. You may also want to fix this in the condition above.